src/shadowbox/docker/Dockerfile

```diff
@@ -25,6 +25,9 @@ LABEL shadowbox.outline-ss-server_version="${SS_VERSION}"
 ARG GITHUB_RELEASE
 LABEL shadowbox.github.release="${GITHUB_RELEASE}"
 
+# Set watchtower to automatically fetch docker image updates
+LABEL com.centurylinklabs.watchtower.enable=true
```

Comment:
On discussion with the team offline, we decided that this is too much of a layering violation. Sorry, could you please move this back to the install script?

Reply:
No worries, moved back to install_server.sh